inc/blocks/namespace.php

```diff
@@ -130,16 +173,21 @@ function on_save_post( int $post_ID, WP_Post $post, bool $update ) : void {
 				'post_content' => serialize_block( $xb ),
 				'post_name' => $xb['attrs']['clientId'],
 				'post_author' => get_current_user_id(),
-				'post_title' => $xb['attrs']['title'] ?: $default_title,
+				'post_title' => $xb['attrs']['title'] ?: $xb_title,
 				'post_parent' => $post_ID,
+				'meta_input' => [
```

Comment:
I didn't know this was a thing to be honest! Might as well move the `_xb_type` meta values here too.

Reply:
I have now added the `_xb_type` meta values here too.